content/en/docs/tasks/administer-cluster/coredns.md

```diff
@@ -58,6 +65,23 @@ customizations after the new ConfigMap is up and running.
 If you are running CoreDNS in Kubernetes version 1.11 and later, during upgrade,
 your existing Corefile will be retained.
 
+## Rollback to kube-dns
+
+From Kubernetes v1.11, Kubeadm will install CoreDNS by default.
+If you want to rollback your cluster to use kube-dns, use the following command:
+
+```
+kubeadm upgrade apply <current-installed-version> --feature-gates CoreDNS=false --force
+```
+This will replace CoreDNS with kube-dns.
+
+Alternately, you can install kube-dns manually with the help of these [instructions](https://github.com/coredns/deployment/tree/master/kubernetes#rollback-to-kube-dns).
+
+**Be advised that there is no translation from the CoreDNS configmap to the kube-dns configmap provided.
+If there is an old kube-dns configmap that had been applied, it will remain and not be deleted.**
```

Comment:
So, do users need to delete the old kube-dns configmap manually first?

Reply:
Kube-dns will optionally mount kube-dns configmap if existed. So if the settings in the old kube-dns configmap work, IMO there is no need to manually delete it first.

Reply:
Agreed. But we need to document this case to users :)

Reply:
Yes, the kube-dns configmap remains undeleted. 
Users can choose to delete it, though it's not needed.